UnitTests/TransportTests.cpp

```diff
@@ -18,6 +19,8 @@
 
 BOOST_AUTO_TEST_SUITE(TransportTests)
 
+using Traits = IPC::UnitTest::Mocks::NullTimeoutTraits;

```

Comment:
But then it's more clear to the reader that, say, IPC::Transport<int, double, InfiniteTimeoutTraits> will never timeout